File: pset2/readability/readability.c
#include <ctype.h>
#include <cs50.h>
#include <stdlib.h>
#include <stdio.h>
#include <string.h>
#include <math.h>

int main (void)
{
    float letterCounter = 0;
    float wordCounter = 1;
    float senCounter = 0;
    string text = get_string("Text: ");
    for (int i = 0; i < strlen(text); i++)
    {
        if(isalpha(text[i]))
        {
         letterCounter++;
        }
        if (text[i] == ' ')
        {
            wordCounter++;
        }
        if (text[i] == '.' || text[i] == '!' || text[i] == '?')
        {
            senCounter++;
        }
    }
    float letters = 100 * (letterCounter / wordCounter);
    float sentences = 100 * (senCounter / wordCounter);

    float index = round(index = 0.0588 * letters - 0.296 * sentences - 15.8);
       // Return result
    if (index < 1)
    {
        printf("Before Grade 1\n");
    }
    else if (index > 16)
    {
        printf("Grade 16+\n");
    }
    else
    {
        printf("Grade %i\n", (int)round(index));
    }
}

File: pset6/readpy/readability.py
from cs50 import get_string
# getting input from user
# initializing variables
text = get_string("Text: ")
lettercount = 0
wordcount = 1
sentencecount = 0
for i in range(len(text)):
    if text[i].isalpha():
        lettercount += 1
    if text[i] == ' ':
        wordcount += 1
    if text[i] == '.' or text[i] == '!' or text[i] == '?':
        sentencecount += 1
letters = 100 * (lettercount / wordcount)
sentences = 100 * (sentencecount / wordcount)
index = round(0.0588 * letters - 0.296 * sentences - 15.8, 2)
index = int(round(index))
if index < 1:
    print("Before Grade 1")
elif index > 16:
    print("Grade 16+")
else:
    print("Grade " + str(index))